.pre-commit-config.yaml

```diff
@@ -15,6 +15,7 @@ repos:
     rev: 5.8.0
     hooks:
       - id: isort
+        args: ["--profile", "black"]
```

Comment:
are you sure isort doesn't look at the pyproject.toml in case it runs through pre commit ?

Reply:
I think so because it wasn't compliant with black... so it was doing a modification then black was doing another modification after wards... so I did this to avoid it

Reply:
argh, yeah ok I thought it actually looked at the pyproject, fine by me then